cypress/e2e/models/administration/credentials/credentials.ts

```diff
@@ -249,7 +248,12 @@ export class Credentials {
 
     edit(cred: CredentialsData): void {
         Credentials.openList();
-        performRowAction(this.name, editAction);
+        cy.get(tdTag, { timeout: 120 * SEC })
```

Comment:
A suggestion:

It seems the change is only in the locator, so instead of re-writing the **`performRowAction`** function in the **`edit`** function, why not define the locator in the constants.ts as ( for example ):
```
export const editActionButton = "#edit-action";
```
and call it as such:
```
performRowAction(this.name, editActionButton);
```

Reply:
No, it is not about locator, it is about changing the way it interacts. It was searching for button with text, now it should search for button by ID. If I will refactor the whole function - it will break functionality of Jira tests. So, it is easier to do what I did